app/migrations/Version20201221175210.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright   <year> Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        https://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\Exception\SkipMigration;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+
+final class Version20201221175210 extends AbstractMauticMigration
+{
+    /**
+     * @throws SkipMigrationException
+     */
+    public function preUp(Schema $schema): void
+    {
+        $shouldRunMigration = false; // Please modify to your needs
+
+        if (!$shouldRunMigration) {
```

Comment:
This migration would never execute queries because it will be skipped due to `$shouldRunMigration` is false.

In prepUp, you should check that new added index are available or not if available then skip.